### ISSUE TITLE
[a11y] Increase some elements from the "Sign in" page

### ISSUE THREAD

**Issue by sblatz** [mozilla/fxa#3989]
## Prerequisite
Make sure you have Google Accessibility Scanner installed.

## Steps to reproduce
1. Launch Fenix and go to Settings - Turn on Sync.
2. Enter a valid email and scan the page.

### Expected behavior
No major discrepancies are displayed.

### Actual behavior
We have 2 major suggestions:
- "Terms of Service" and "Privacy Notice" are suggested to be increased from 15dp to 48dp or larger,
- "Use a different account" and "Forgot password?" are suggested to be increased from 19dp to 48dp or larger.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8), Samsung Galaxy Note 8 (Android 9)
* Fenix version: RC 3.0.2, Beta build 3.1.0-beta 3, Nightly build from 01/15

![sync](https://user-images.githubusercontent.com/40694158/72442220-c8291d00-37b4-11ea-87d1-0d79b5b5f6b2.png)

First filed in the Fenix repo: https://github.com/mozilla-mobile/fenix/issues/7722

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/FXA-990)
┆Issue Number: FXA-990

**Comment by LZoog** [comment]
@ryanfeeley I need your recommendation here since a true fix for this would affect the design fairly significantly.

I increased the padding around one of the links to see what increasing the clickable area would look like and outlined it in a hotpink border to visualize it. We would want to increase the space above/below all the links/button to prevent users from accidentally clicking something they don't mean to if we went this route (so we'd need more space between sign in/the text below for example), another accessibility concern.

 The WCAG does not appear to have official, specific measures around clickable content size. I could definitely see the argument for adding a `5px` or so padding on the top/bottom part of our links, but this (`17px`) seems excessive, and I'd argue the tool used to suggest this is simply incorrect as far as standard ADA compliance goes.

So Ryan, what do you think - do we want to silence the error and make the suggested fix (not recommended, it'd greatly impact design and the WCAG does not require it), do a half-measure fix by adding a small amount of top/bottom padding which shouldn't affect the design but may in some cases since it'll be a global small change on our links, or mark as a `won't fix` since this does not make us fail with other testing tools?

![image](https://user-images.githubusercontent.com/13018240/73769710-bba73d00-4740-11ea-976e-d1920b366b47.png)

**Comment by ryanfeeley** [comment]
We will be revisiting fxa auth screens in future to apply a vertical grid and will take this into account. Like most links in the body some users will likely need to zoom in to tap specific links.
